### PR TITLE
added post-highlight-section-1

### PR DESCRIPTION
**Pull request by spriha1301**
post highlight section-1 desktop view
highlight

### CODE REVIEW

**Comment by spriha1301** [comment]
mobile view1

File: blogs/01/index.html [hunk 1]
```diff
@@ -18,51 +18,40 @@ <h3>Mini Blog</h3>
...
     </header>
-  
-    <section class="post-highlight-section-1">
-        <div class="highlight-section-wrapper">
+          
+        <section class="post-highlight-section-1">
+         <div class="post-highlight">
+         
+             <div class="box-section">
+               <div class="box-1"> <img src= "https://placehold.co/345x200/gray/white" placeholder=""></div>
```
Comment (itsashishbharti) [inline blogs/01/index.html:37]: fix the all  images size .
get reference from the original site .

File: blogs/01/index.html [hunk 2]
```diff
@@ -18,51 +18,40 @@ <h3>Mini Blog</h3>
...
 
-            <div class="card-section-1">
-                    <div class="grid-box grid-box-1">  <img src="https://placehold.co/345x200/gray/white" placeholder=""></div>
-                    <div class="grid-box">  <img src= "https://placehold.co/345x200/orange/white" placeholder=""> </div>      
-            </div>
-        
+
+              <div class="box-section"> 
+               <div class="box-1"> <img src= "https://placehold.co/345x200/gray/white" placeholder=""></div>
+               <div class="box-2">  <img src= "https://placehold.co/345x200/black/white" placeholder=""></div>
```
Comment (itsashishbharti) [inline blogs/01/index.html:46]: missing some text content inside images. fixed it in every images

File: blogs/01/styles/post-highlight-section-1.css [hunk 3]
```diff
@@ -1,36 +1,35 @@
 section.post-highlight-section-1{
-    background-color: #f8f9fa;
+  background-color:#f8f9fa;
```
Comment (itsashishbharti) [inline blogs/01/styles/post-highlight-section-1.css:2]: use variable name place of color code.

File: blogs/01/index.html [hunk 4]
```diff
@@ -18,51 +18,40 @@ <h3>Mini Blog</h3>
...
+          
+        <section class="post-highlight-section-1">
+         <div class="post-highlight">
+         
+             <div class="box-section">
+               <div class="box-1"> <img src= "https://placehold.co/345x200/gray/white" placeholder=""></div>
+               <div class="box-2">  <img src= "https://placehold.co/345x200/black/white" placeholder=""></div> 
+           </div>
+           
+            <div class="highlight"> <img src= "https://placehold.co/345x200/yellow/white" placeholder=""></div>
```
Comment (itsashishbharti) [inline blogs/01/index.html:41]: change the size of that also and inside image add element according to reference site.

File: blogs/01/styles/post-highlight-section-1.css [hunk 5]
```diff
@@ -1,36 +1,35 @@
...
         margin: 0px auto;
         height: 1000px; 
     }
-    div.highlight-section-wrapper>div.card-section-1>div.grid-box-1{
-        margin-top: 35px;
+    div.post-highlight>div.box-section>div.box-1{
+        margin-top: 30px;
     }
-    div.voucher{
+    div.boxes{
```
Comment (itsashishbharti) [inline blogs/01/styles/post-highlight-section-1.css:32]: why you use this div class name .
you are not decleared that div or class of that section of html

**Comment by amezng** [comment]
not as per the design

**Pull request closed without merging**